modules/nlu/src/test/types.ts

```diff
@@ -0,0 +1,25 @@
+import { IO } from 'botpress/sdk'
+
+/**
+ * @description extending IO.EventUnderstanding as is is necessary in this context (nlu/backend/election) as the Typescript compiler doesn't allow me to import botpress/sdk in a test file
+ * @todo fix Typescript configuration file to allow botpres/sdk import in test files
```

Comment:
**PP**

A comment like 

```ts
// This function is located here because importing the SDK does not build in test files
```

would do the job

Reply:
Done